### PR TITLE
Import structures/mobile & fix some styles

### PR DESCRIPTION
**Pull request by d2bit**
captura de pantalla 2015-05-08 a las 14 03 47

### CODE REVIEW

**Comment by gponsu** [comment]
merge it! :)

**Comment by Fernan2** [comment]
Still some styles should be fixed... "Entrar" needs more contrast

**Comment by d2bit** [comment]
Styles should be applied in the main projects as config variables

**Comment by maxvlc** [comment]
:+1:

File: app/assets/stylesheets/psique/foundation/_tools.scss [hunk 1]
```diff
@@ -18,11 +18,25 @@
 //   Margin
 // -------------------------------------
 
+.mt-1 { margin-top: 1rem; }
 .mtb-1 {
-  margin-top: 1em;
-  margin-bottom: 1em;
+  margin-top: 1rem;
+  margin-bottom: 1rem;
```
Comment (Dygerydoo) [inline app/assets/stylesheets/psique/foundation/_tools.scss]: This may be margin: 1rem 0; to avoid the use of two properties no?

Comment (d2bit) [inline app/assets/stylesheets/psique/foundation/_tools.scss]: I don't want to override left and right margins.

Comment (Fernan2) [inline app/assets/stylesheets/psique/foundation/_tools.scss]: maybe `margin: 1rem inherit;`? but for me it's also good as is...

**Comment by Fernan2** [comment]
what about my previous `"Entrar" needs more contrast` comment?

**Pull request merged**